.devcontainer/devcontainer.json

```diff
@@ -1,3 +1,36 @@
 {
-  "onCreateCommand": "curl -L https://foundry.paradigm.xyz | bash && /home/codespace/.foundry/bin/foundryup && pnpm install"
+  "onCreateCommand": "curl -L https://foundry.paradigm.xyz | bash && /home/codespace/.foundry/bin/foundryup && pnpm install",
+  "name": "latticexyz/mud Dev Container",
+  "features": {
+    "ghcr.io/devcontainers/features/git": {},
+    "ghcr.io/devcontainers/features/github-cli": {}
+  },
+  // TODO which other ports do we include here?
+  "forwardPorts": [
+    8545
+  ],
+  "portsAttributes": {
+    "8545": {
+      "label": "Foundry Anvil",
+      "onAutoForward": "notify"
+    }
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "JuanBlanco.solidity",
+        "dbaeumer.vscode-eslint",
+        "esbenp.prettier-vscode"
+      ],
+      "settings": {
+        "solidity.monoRepoSupport": true,
+        "npm.packageManager": "pnpm",
+        // TODO get correct version
+        "solidity.compileUsingRemoteVersion": "v0.8.17",
```

Comment:
is it possible to have "logic" in this json that grabs the solidity version from somewhere?